docs/Refund.md

```diff
@@ -0,0 +1,29 @@
+# Refunds
+
+Refunds are one of the applications built on top of the `Pull Payments` feature.
```

Comment:
Can we link to pull payments instead? It's good to have links from one page to another, sorry , misted this one earlier.

Reply:
No problem ! 
```suggestion
Refunds are one of the applications built on top of the [Pull Payments](./PullPayments.md) feature.
```